qa/setup_packages.py

```diff
@@ -263,6 +273,40 @@ def max_cuda_version(self, cuda_version):
                 max_cuda = ver
         return max_cuda
 
+
+class CudaPackageExtraIndex(CudaPackage):
+    """Class describing a cuda package with a key/name and a dictionary where the key
+        is a cuda version and value is the list of versions supported.
```

Comment:
```suggestion
        is a cuda version and value is the list of versions supported.
```

Reply:
?

Reply:
nevermind, I edited it back (thought the suggestion will be removed)

Reply:
Ok